src/cp/cp_control_point.c

```diff
@@ -344,25 +312,20 @@ __wt_session_control_point_init_all(WT_SESSION_IMPL *session)
 int
 __wt_conn_control_point_enable_all_in_open(WT_CONNECTION_IMPL *conn, const char **cfg)
 {
-#if 1 /* If no per connection control points are enabled at the start. */
-    WT_UNUSED(conn);
-    WT_UNUSED(cfg);
-#else
     WT_CONTROL_POINT_REGISTRY *control_points;
+    int idx;
 
+    idx = 0;
     if (CONNECTION_CONTROL_POINTS_SIZE == 0)
         return (0);
     control_points = conn->control_points;
 
-    /*
-     * This part must be edited. Repeat this for every per connection control point that starts
-     * enabled.
-     */
-#if 0 /* For example. */
-    WT_RET(__wti_conn_control_point_enable(
-      conn, &(control_points[WT_CONN_CONTROL_POINT_ID_EXAMPLE1]), cfg));
-#endif
-#endif
+    for (; idx < CONNECTION_CONTROL_POINTS_SIZE; idx++) {
+        if (control_points[idx].enable_at_open == false)
+            continue;
```

Comment:
Delete these lines. The correct value is defined above for each control point based upon the ControlPoint enable_in_open value.